cli/cmd/get_cmd.go

```diff
@@ -35,6 +35,10 @@ var getCmd = &cobra.Command{
 			return "", err
 		}
 
+		if output == string(formatters.JSON) {
+			ctx = context.WithValue(ctx, "X-Tracetest-Augmented", true)
+		}
+
```

Comment:
Is there a reason to propagate this value in a context (like propagating it deeper on the call stack)? We could use the options pattern to pass this value.

Reply:
This was the easiest way to propagate the value. The problem with the option would be that we would change the interface of all the client methods.

Reply:
but we can do it if you think that's better